_about/notes/0.8.md

```diff
@@ -8,4 +8,35 @@ type: markdown
 ---
 {% include home.html %}
 
-FILL IN BEFORE RELEASE
+In addition to the usual pile of bug fixes and performance improvements, this release includes the new or
+updated features detailed below.
+
+## Networking
+
+- **Revamped Traffic Management Model**. We're finally ready to take the wraps off our new traffic
+management configuration model. The new model adds many new features and addresses usability issues
+with the prior model.
+[Learn more]({{home}}/docs/tasks/traffic-management-v1alpha3/)
+
+- **IPv6**. [TBD: do we want to say anything about IPv6 support?]
+
+- **Envoy V2**. [TBD: should we say anything about all the work to use the Envoy V2 API?
+Does this have any user-visible effect?]
+
+## Security
+
+- **Multicluster Support**. [TBD: What should we say?]
+
+- **Authentication Filter**. [TBD: What should we say about the proxy authn filter?]
+
```

Comment:
Should we add per namespace auth policy (mTLS enablement) here?   cc @wattli

Reply:
We should also mention the new name citadel.

Reply:
**Authentication Policy**. We support setup mutual TLS and end-user authentication using authentication policy.